source/slang/slang-ast-base.h

```diff
@@ -2,25 +2,26 @@
 
 #pragma once
 
-#include "slang-ast-reflect.h"
+#include "slang-ast-base.h.fiddle"
```

Comment:
What is being generated in slang-ast-base.h.fiddle?

Reply:
`slang-ast-base.h` has the class declarations for `NodeBase`, `Type`, `Val`, etc., and all of the `NodeBase`-derived classes need to have the `FIDDLE(...)` macro invocations in their bodies replaced with the auto-generated declarations for that type.